website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
 
-`talosctl` is a CLI tool which interfaces with the Talos API.
-Talos Linux has no SSH access: `talosctl` is the tool you use to interact with the operating system on the machines.
 
-You can download `talosctl` on MacOS and Linux via:
+    You can install `talosctl` on macOS or Linux by running:
 
 ```bash
 brew install siderolabs/tap/talosctl
 ```
 
-For manual installation and other platforms please see the [talosctl installation guide]({{< relref "../talos-guides/install/talosctl.md" >}}).
+Refer to the [talosctl installation guide](/v1.10/talos-guides/install/talosctl) for installation on other platforms.
+    
+- **Ensure network access**: Your machines will need internet access to download the Talos installer and container images, sync time, and more.
 
-> Note: If you boot systems off the ISO, Talos on the ISO image runs in RAM and acts as an installer.
-> The version of `talosctl` that is used to create the machine configurations controls the version of Talos Linux that is installed on the machines - NOT the image that the machines are initially booted off.
-> For example, booting a machine off the Talos 1.3.7 ISO, but creating the initial configuration with `talosctl` binary of version 1.4.1, will result in a machine running Talos Linux version 1.4.1.
->
-> It is advisable to use the same version of `talosctl` as the version of the boot media used.
 
-### Network access
+    If you’re working in a restricted network environment, check out the official documentation on using [registry proxies](/v1.10/talos-guides/configuration/pull-through-cache), local registries, or setting up an [air-gapped installation](/v1.10/advanced/air-gapped).
 
-This guide assumes that the systems being installed have outgoing access to the internet, allowing them to pull installer and container images, query NTP, etc.
-If needed, see the documentation on [registry proxies]({{< relref "../talos-guides/configuration/pull-through-cache" >}}), local registries, and [airgapped installation]({{< relref "../advanced/air-gapped" >}}).
+## Talos Cluster Setup Overview
 
-## Acquire the Talos Linux image and boot machines
+Every Talos cluster follows the same process, regardless of where you deploy it:
 
-The most general way to install Talos Linux is to use the ISO image.
+1. **Boot** - Start machines with the Talos Linux image
+2. **Configure** - Create a root of trust certificate authority and generate configuration files
+3. **Apply** - Apply machine configurations to the nodes
+4. **Connect** - Set up your local `talosctl` client
+5. **Bootstrap** - Initialize the Kubernetes cluster. 
 
-The latest ISO image can be found on the Github [Releases](https://github.com/siderolabs/talos/releases) page:
+**Note**: You can also opt to use the Omni GUI to create a Talos cluster spanning different platforms, bare metal, cloud providers, and virtual machines.
```

Comment:
There should be a link leading to omni here. Also the term "GUI" is incorrect as omni is much more than a GUI.